### ISSUE TITLE
Libre Calc Issues from the Year 2062

### ISSUE THREAD

**Issue by asusman** [LucisVivae/Australian-Fire-Calculator-excel#1]
First off thanks for sharing this great spreadsheet. Its fantastic and much appreciated. 

I have an issue using LibreCalc and cant figure it out and was hoping for some help. 

Its a brand new vanilla spreadsheet and it seems to work well until the year 2062 when the chart stops working. 

I have troubleshooted the first error to Calculations > Column Q > Row 2114. The error is Err:504 and the spreadsheet errors out after and impacts all columns from P to BB. 

I have found that unlocking the sheet and deleting the contents on Column Q resolves the issue and the chart then continues to 2081, 
but I haven't been able to find why the formulas are failing when there is data in that column.  

Are you able to assist?

**Comment by LucisVivae** [comment]
I unfortunately don't have libre calc (I tried using the portable version last night and the file refused to open. However, I am 99% sure based on the error provided that this issue is because to save file size (saves about 2mb) I only did the Loan calculations for 50 years based on mortgages being paid out before then.

If you go to the Loan_Calc sheet select the entire last row and then drag that to row 5221 does that fix the issue for you?
